__tests__/Action.test.ts

```diff
@@ -209,34 +211,9 @@ describe("Action", () => {
             expect(error).toEqual("error")
         }
 
-        expect(getMock).toBeCalledWith(tag)
-        expect(updateMock).not.toBeCalled()
-        expect(uploadMock).not.toBeCalled()
-    })
-
-    it("throws error when list has no data", async () => {
```

Comment:
Could you bring this test back? Guessing you originally removed it when you removed the corresponding code from Action (but we reverted that).

Reply:
Yep, sorry I missed that